modular-particle-system/src/initializers/randomRotationalVelocity.ts

```diff
@@ -0,0 +1,57 @@
+import { Module, ModuleObject } from "../module";
+import { loadSerializedProperty, deserializePrimitiveDataType } from "../serialization/moduleSerialization";
+import { Particle } from "../particle";
+import { ParticleEffect } from "../particleEffect";
+import { lerp } from "../utilities";
+
+/**
+ * Module that assigns each particle random _rotational velocity_.
+ * This causes the particle to continuously rotate with a static speed along its lifetime.
+ *
+ * Rotational velocity range can be customized with `min` and `max` properties.
+ */
+export class RandomRotationalVelocity extends Module {
+    /**
+     * Min rotational velocity as **Radians per second**
+     */
+    min = -2 * Math.PI;
```

Comment:
can we rename it to minValue or minVelocity ?

Reply:
I think it was originally an idea from Risto to have property names such as this min, max. They are thought from the view of looking most logical when they are used in code, like for example:

```
const rotationVelocity = new RandomRotationalVelocity(effect)
effect.modules.push(rotationVelocity)
rotationVelocity.min = 5
```

And for reference the suggested name:

```
rotationVelocity.minVelocity = 5
rotationVelocity.minValue = 5
```

Risto's original point was that it doesn't look nice to have this kind of `velocity.velocity = something` code.

What do you think about this?

Reply:
Well, if Risto thinks this way, then we can keep what it is. Then I will mark it approved.